### PR TITLE
added docker file

### PR DESCRIPTION
**Pull request by felixzhu17**
docker build -t python-relevanceai .

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
```
Comment (boba-and-beer) [inline Dockerfile:1]: hey who'd you choose python3.8

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+ADD . /relevanceai/
+WORKDIR /relevanceai/
+ 
+RUN python setup.py install
```
Comment (boba-and-beer) [inline Dockerfile:6]: pip install -e .[viz] or something. It should have everything if possible

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+CMD ["python"]
```
Comment (boba-and-beer) [inline Dockerfile:8]: nah no need for this I reckon

**Review by boba-and-beer — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (felixzhu17) [reply Dockerfile:1]: which python do you want?

Comment (boba-and-beer) [reply Dockerfile:1]: 3.7 for now please

**Comment by boba-and-beer** [comment]
- Missing .dockerignore 
- Not sure where docker container is hosted 
- Missing build instructions in README
- Missing Jupyter notebook env if possible 

Closing for now. Will return to this later.

**Pull request closed without merging**